### PR TITLE
Adds xmp_write feature to make xmp-toolkit inclusion optional

### PR DESCRIPTION
**Pull request by gpeacock**
## Changes in this pull request
Adds xmp_write feature to make xmp-toolkit inclusion optional.
The dc:provenance XMP data will not be written unless this feature is enabled.

## Checklist
- [ ] This PR represents a single feature, fix, or change.
- [ ] All applicable changes have been documented.
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment.

### CODE REVIEW

File: sdk/Cargo.toml [hunk 1]
```diff
@@ -14,6 +14,7 @@ rust-version = "1.58.0"
 [features]
 async_signer = ["async-trait"]
 file_io = ["openssl"]
+xmp_write = ["xmp_toolkit"]
```
Comment (scouten-adobe) [inline sdk/Cargo.toml:17]: Please also add this feature to the README.

**Review by scouten-adobe — CHANGES_REQUESTED** [review]
Aside from the requested addition to README, :+1:.

**Review by mauricefisher64 — APPROVED** [review]
👍

File: sdk/Cargo.toml (hunk 1 above)
Comment (gpeacock) [reply sdk/Cargo.toml:17]: Added a readme line for the  xmp_write feature

**Review by scouten-adobe — APPROVED** [review]

**Pull request merged**